component/azstorage/config.go

```diff
@@ -385,7 +388,18 @@ func ParseAndValidateConfig(az *AzStorage, opt AzStorageOptions) error {
 	az.stConfig.cancelListForSeconds = opt.CancelListForSeconds
 
 	az.stConfig.telemetry = opt.Telemetry
-
+	az.stConfig.blobFilter = opt.BlobFilter
+	if len(opt.BlobFilter) > 0 {
+		fmt.Println(opt.BlobFilter)
```

Comment:
You can move fmt.Println to log.Debug as blobfuse will have no console attached to print,